knowledge-base/grid-autofit-columns.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Autofit all Grid columns on the initial load. 
+description: Autofit all Grid columns on the initial load. 
+type: troubleshooting
+page_title: Autofit all Grid columns on the initial load. 
+slug: grid-autofit-columns-on-initial-load
+position: 
+tags: 
+res_type: kb
+---
+
+
+## Description
+
+I would like to autofit the Grid column on the initial load of the Component using the [AutoFitAllColumns]({%slug components/grid/columns/resize%}#autofit-columns) method of the Grid reference.
+
+#### Problematic behavior
```

Comment:
We can present the information in a positive way. No need for "problematic" opinions.
```suggestion
## Background
```